Makefile

```diff
@@ -1,12 +1,13 @@
 .PHONY: build configure prepare clean
 
 build: configure
-	docker run --rm -t --privileged -v $(PWD)/vyos-build:/vyos -w /vyos vyos/vyos-build:current sudo make iso
+	docker run --rm -t --privileged -v $(PWD)/vyos-build:/vyos -w /vyos vyos/vyos-build:current sudo ./build-vyos-image iso --architecture amd64 --custom-apt-key ../tailscale.gpg --custom-apt-entry "deb https://pkgs.tailscale.com/stable/debian bullseye main" --custom-package "tailscale" --build-comment "VyOS with Tailscale" --build-type release --version 1.4-rolling-`date +%Y%m%d%H%M`
 	mkdir -p ./build
 	mv vyos-build/build/vyos-*.iso ./build
 
 configure: prepare
-	docker run --rm -t --privileged -v $(PWD)/vyos-build:/vyos -w /vyos vyos/vyos-build:current ./configure --architecture amd64 --custom-apt-key ./tailscale.gpg --custom-apt-entry "deb https://pkgs.tailscale.com/stable/debian bullseye main" --custom-package "tailscale" --build-comment "VyOS with Tailscale" --build-type production --version 1.4-rolling-`date +%Y%m%d%H%M`
+	docker run --rm -t --privileged -v $(PWD)/vyos-build:/vyos -w /vyos -e os=buster64 -e branch=sagitta vyos/vyos-build:current make build
```

Comment:
This make target doesn't seem to exist as far as I can tell?https://github.com/vyos/vyos-build/blob/current/Makefile

Reply:
Oh yeah! I think it can. It's mentioned here and I think now looking at it again it's _just_ for macOS: https://docs.vyos.io/en/latest/contributing/build-vyos.html#build-iso

Reply:
I had a dig into this (finally) and realised we can drop all of this part of our `Makefile` as the new build script does a clean itself first. I've pushed up a commit which should resolve this.